flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackendConfigTest.java

```diff
@@ -481,10 +479,6 @@ public void testCallsForwardedToNonPartitionedBackend() throws Exception {
 						env.getTaskKvStateRegistry());
 	}
 
-	static Environment getMockEnvironment() {
-		return getMockEnvironment(new File[] { new File(System.getProperty("java.io.tmpdir")) });
-	}
-
 	static Environment getMockEnvironment(File[] tempDirs) {
```

Comment:
we could change this to a vararg method; then we could simplify calls like `getMockEnvironment(new File[] { tempFolder.newFolder() });`.